Src/FluentAssertions/Equivalency/EquivalencyAssertionOptionsBuilder.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq.Expressions;
+using FluentAssertions.Common;
+using FluentAssertions.Equivalency.Selection;
+
+namespace FluentAssertions.Equivalency
+{
+    public class EquivalencyAssertionOptionsBuilder<TExpectation, TCurrent> : EquivalencyAssertionOptions<TExpectation>
```

Comment:
🔧 I think something like `NestedExclusionOptionBuilder` would better convey the purpose of this class.

Reply:
✅ Renamed to `NestedExclusionOptionBuilder`